### ISSUE TITLE
Support Chinese new infinity server?

### ISSUE THREAD

**Issue by zjh4473** [pyfa-org/Pyfa#2471]
just like the question.

some new features and new ships are added on infinity server, does pyfa support this server?

**Comment by DarkFenX** [comment]
No. Pyfa does not support the server and there are no plans to (or, I can paraphrase, I have no intention to as the only maintainer of the project).

However, if there are any volunteers who play on Infinity, I can explain the process of updating pyfa, and work out a process to keep codebase more or less compatible between servers.

**Comment by zjh4473** [comment]
> No. Pyfa does not support the server and there are no plans to (or, I can paraphrase, I have no intention to, as the only maintainer of the project). It's about the effort, I already struggle to find time to maintain it for TQ.
> 
> However, if there are any volunteers who play on Infinity, I can explain the process of updating pyfa, and work out a process to keep codebase more or less compatible between servers.

if i may, i think i can do that. (if pyfa core is not too complicate)
the difficult part is the neural network cloud computing feature.
anyway, if you need help on this, im happy to assist.

**Comment by DarkFenX** [comment]
Do you have any non-chinese-specific instant messenger? I have no idea what's going on infinity (apart from some of initial features).

I am currently busy with some IRL & EVE stuff, but hopefully we will find some time to discuss.

**Comment by zjh4473** [comment]
of course, my tg is https://t.me/Name_is_Tony
